src/gurt/shm_dict.c

```diff
@@ -669,6 +669,7 @@ shm_ht_rec_find_insert(d_shm_ht_loc_t shm_ht_loc, const char *key, const int len
 	d_shm_ht_rec_t  rec_next = NULL;
 	char           *value    = NULL;
 	int             rc;
+	int             padding;
```

Comment:
the padding fix here sounds unrelated to this PR and should be split into a different PR?